From: Ben Hutchings <ben@decadent.org.uk>
Date: Sat, 21 May 2011 18:12:37 +0100
Subject: [PATCH 5/6] dma-mapping.h: add the dma_unmap state API

For 2.6.32, define the generic API using the PCI API instead of the
other way around as in 0acedc124aca35f5cce9d4ee288dc372bf517e09.
---
 include/linux/dma-mapping.h |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/include/linux/dma-mapping.h b/include/linux/dma-mapping.h
index 8357721..78211a0 100644
--- a/include/linux/dma-mapping.h
+++ b/include/linux/dma-mapping.h
@@ -240,4 +240,11 @@ struct dma_attrs;
 
 #endif /* CONFIG_HAVE_DMA_ATTRS */
 
+#define DEFINE_DMA_UNMAP_ADDR(ADDR_NAME) DECLARE_PCI_UNMAP_ADDR(ADDR_NAME);
+#define DEFINE_DMA_UNMAP_LEN(LEN_NAME)   DECLARE_PCI_UNMAP_LEN(LEN_NAME);
+#define dma_unmap_addr             pci_unmap_addr
+#define dma_unmap_addr_set         pci_unmap_addr_set
+#define dma_unmap_len              pci_unmap_len
+#define dma_unmap_len_set          pci_unmap_len_set
+
 #endif
-- 
1.7.5.4

